### PR TITLE
Updates all codebase uri's to say StateVoicesNational

### LINKED ISSUE #2321: Transition all `MoveOnOrg` links in codebase to point towards `StateVoicesNational`

### PR DESCRIPTION
**Pull request by Arique1104**
# Fixes #2321 

## Description
This pull request combs through all references of MoveOnOrg in the documentation and codebase to point to StateVoicesNational.

- 101 results of MoveOnOrg uri within 26 files

# Checklist:

- [ ] I have manually tested my changes on desktop and mobile
- [ ] The test suite passes locally with my changes
- [ ] If my change is a UI change, I have attached a screenshot to the description section of this pull request
- [ ] [My change is 300 lines of code or less](https://github.com/MoveOnOrg/Spoke/blob/main/CONTRIBUTING.md#submitting-your-pull-request), or has a documented reason in the description why it’s longer
- [ ] I have made any necessary changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] My PR is labeled [WIP] if it is in progress

### CODE REVIEW

**Comment by Arique1104** [comment]
- Purposefully did NOT edit a migration file: `Spoke/migrations/20190207220000_init_db.js` 
- This document holds a commented out foot note.  It is used a sudo-code reference.
- Waiting for approval from engineering team.

**Review by dwbond — APPROVED** [review]
Looks great to me!

**Pull request merged**
